components/server/src/authorization/spicedb-authorizer.ts

```diff
@@ -119,6 +123,20 @@ export class SpiceDBAuthorizerImpl implements SpiceDBAuthorizer {
     }
 
     async writeRelationships(...updates: v1.RelationshipUpdate[]): Promise<v1.WriteRelationshipsResponse | undefined> {
+        const result = await this.writeRelationshipsInternal(...updates);
+        const writtenAt = result?.writtenAt?.token;
+        await this.tokenCache.set(
+            ...updates.map<ZedTokenCacheKV>((u) => [
+                u.relationship?.resource,
+                writtenAt, // Make sure that in case we don't get a writtenAt token here, we at least invalidate the cache
```

Comment:
I would always expect a token on successful writes. Did you see cases where no writtenAt was returned?

Reply:
The API gives no guarantee, so I decided to not assume things. :shrug:

Reply:
I doubt it happens, but let's at least log a warning in those cases